### PR TITLE
WYSIWYG: Override "Link to a file" functionality to support cloudinary

### PR DESCRIPTION
**Pull request by lbradley-mm**

### CODE REVIEW

File: client/src/js/TinyMCE_ssmedia.js [hunk 1]
```diff
@@ -94,7 +108,41 @@
...
+        /**
+         * Inserts the data into the editor as a link
+         *
+         * @param {*} response
+         */
+        linkHandler(response) {
+            const asset = response.assets[0];
+
+            let secureUrl;
+            if (asset.derived && asset.derived.length > 0) {
```
Comment (mrai-mm) [inline client/src/js/TinyMCE_ssmedia.js:122]: For the sake of readability, can you put a newline before and after if/else statements?

File: client/src/js/TinyMCE_ssmedia.js [hunk 2]
```diff
@@ -94,7 +108,41 @@
...
+                secureUrl = asset.derived[0].secure_url;
+            } else {
+                secureUrl = asset.secure_url;
+            }
+            let linkText = this.editor.selection.getContent({format: 'html'});
+            if (linkText.trim().length <= 0) {
```
Comment (mrai-mm) [inline client/src/js/TinyMCE_ssmedia.js:128]: Same as above.

File: client/src/js/TinyMCE_ssmedia.js [hunk 3]
```diff
@@ -94,7 +108,41 @@
...
+                linkText = prompt('Link text');
+            }
+
+            let defaultDescription;
+            if (
```
Comment (mrai-mm) [inline client/src/js/TinyMCE_ssmedia.js:133]: Same as above.

**Review by mrai-mm — CHANGES_REQUESTED** [review]

File: client/src/js/TinyMCE_ssmedia.js (hunk 2 above)
Comment (lbradley-mm) [reply client/src/js/TinyMCE_ssmedia.js:130]: Done

File: client/src/js/TinyMCE_ssmedia.js (hunk 1 above)
Comment (lbradley-mm) [reply client/src/js/TinyMCE_ssmedia.js:123]: Done

**Pull request merged**